Hm, doh - looks like you need to define scaleQuadruple in src/track/beatmap.h around line 103.
```
[CXX] src\track\beatmap.cpp
beatmap.cpp
src\track\beatmap.cpp(576) : error C3861: 'scaleQuadruple': identifier not found
src\track\beatmap.cpp(626) : error C2039: 'scaleQuadruple' : is not a member of 'BeatMap'
        C:\projects\mixxx\src\track/beatmap.h(22) : see declaration of 'BeatMap'
src\track\beatmap.cpp(627) : error C2065: 'm_beats' : undeclared identifier
src\track\beatmap.cpp(627) : error C2228: left of '.first' must have class/struct/union
        type is 'unknown-type'
src\track\beatmap.cpp(629) : error C2065: 'm_beats' : undeclared identifier
src\track\beatmap.cpp(629) : error C2228: left of '.begin' must have class/struct/union
        type is 'unknown-type'
src\track\beatmap.cpp(630) : error C2065: 'm_beats' : undeclared identifier
src\track\beatmap.cpp(630) : error C2228: left of '.end' must have class/struct/union
        type is 'unknown-type'
src\track\beatmap.cpp(634) : error C2065: 'i' : undeclared identifier
src\track\beatmap.cpp(634) : error C2143: syntax error : missing ';' before ')'
src\track\beatmap.cpp(635) : error C2065: 'i' : undeclared identifier
src\track\beatmap.cpp(636) : error C2065: 'm_beats' : undeclared identifier
src\track\beatmap.cpp(636) : error C2228: left of '.insert' must have class/struct/union
        type is 'unknown-type'
scons: *** [win32_build\track\beatmap.obj] Error 2
scons: building terminated because of errors.
```

--- a/src/track/beatmap.cpp
+++ b/src/track/beatmap.cpp
@@ -631,7 +631,7 @@ void BeatMap::scaleQuadruple() {
         // Need to not accrue fractional frames.
         int distance = it->frame_position() - prevBeat.frame_position();
         Beat beat;
-        for (i=1; i<=3, i++) {
+        for (i=1; i<=3, i++;) {
             beat.set_frame_position(prevBeat.frame_position() + distance * i / 4);
             it = m_beats.insert(it, beat);
         }

--- a/src/track/beatmap.cpp
+++ b/src/track/beatmap.cpp
@@ -631,7 +631,7 @@ void BeatMap::scaleQuadruple() {
         // Need to not accrue fractional frames.
         int distance = it->frame_position() - prevBeat.frame_position();
         Beat beat;
-        for (i=1; i<=3, i++;) {
+        for (int i=1; i<=3, i++;) {
             beat.set_frame_position(prevBeat.frame_position() + distance * i / 4);
             it = m_beats.insert(it, beat);
         }

--- a/src/track/beatmap.h
+++ b/src/track/beatmap.h
@@ -98,6 +98,7 @@ class BeatMap : public QObject, public Beats {
 
     void scaleDouble();
     void scaleTriple();
+    void scaleQuadruple();
     void scaleHalve();
     void scaleThird();
     void scaleFourth();

